websites/D/Discord SFX/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "957840712404193290",
+		"name": "can"
+	},
+	"service": "Discord SFX",
+	"description": {
+		"en": "Discord SFX lists some sound files uploaded by other site users that you can download and use for the soundboard of your Discord server, and also allows you to share your own."
+	},
+	"url": "discordsfx.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/T6CpbXk.png",
```

Comment:
May be difficult to see for light-mode users. Consider adding the background color.